TACC/Core-Portal
# Task/fp 318 job history modal  ui tweaks

# Notice: #

Keeping as Draft. See https://github.com/TACC/Frontera-Portal/pull/99#pullrequestreview-462385586.

- `task/FP-318-job-history-modal` #99
    - `task/FP-318-job-history-modal--ui-tweaks-more-date-expand-tweaks` #143
        - `task/FP-318-job-history-modal--ui-tweaks` #140
        - `task/FP-318-job-history-modal--expand-tweaks` #141
        - `task/FP-318-job-history-modal--more-data` #142
        - `task/FP-318-job-history-modal--re-style` #144

## Overview: ##

_(via `task/FP-318-job-history-modal`; see PR #99)_

UI tweaks.

Add input data that is not shown.

## PR Status: ##


## Related Jira tickets: ##

* [FP-318](https://jira.tacc.utexas.edu/browse/FP-318)

## Summary of Changes: ##

_(each change is a single commit)_

- Remove excess `<dl>` wrap markup.
- Update `…Link` component to accept text as children (like other `…Link` components).
- Move "FailureReport" into "Status".
    Change "FailureReport": "Failure Report" to "Failure Report": "System Output".
- Remove duplicate CSS.
- Use `<DefinitionList>` component for left panel, also.
- Use appropriate `pre-wrap` formatting for "Failure Report" status message.

## Testing Steps: ##
1. Open Job History Detail View modal.
2. Ensure Expand panel still works.
3. Ensure left panel and right panel layout matches target branch.
4. Ensure left panel and right panel layout both use `<dl>`.
5. Open "System Output" to see last status message with proper formatting.
6. Ensure other styles match target branch.

## UI Photos:

<img width="806" alt="FP-318 UI Tweaks" src="https://user-images.githubusercontent.com/62723358/89519644-4f1a0300-d7a2-11ea-9fa8-aea408f74e4c.png">
<img width="806" alt="FP-318 UI Tweaks - Panel Open" src="https://user-images.githubusercontent.com/62723358/89519640-4d503f80-d7a2-11ea-84c5-535f325af253.png">

## Notes: ##
